packages/angular_devkit/build_angular/src/builders/dev-server/vite-server.ts

```diff
@@ -216,8 +216,15 @@ export async function* serveWithVite(
       );
 
       server = await createServer(serverConfiguration);
-
       await server.listen();
+
+      if (browserOptions.ssr) {
+        // Vite will only start dependency optimization of SSR modules when the first request comes in.
+        // In some cases, this causes a long waiting time. To mitigate this, we call `ssrLoadModule` to
+        // initiate this process before the first request.
+        void server.ssrLoadModule('<deps-caller>').catch(() => {});
```

Comment:
Can you add a note here that this will intentionally fail from the unknown module argument but there is no other way currently to initiate the ssr dep optimizer?